dashboard/app/models/script_level.rb

```diff
@@ -94,7 +94,7 @@ def next_level_or_redirect_path_for_user(user)
     # to unplugged level (example: if you start a sequence of
     # assessments associated with an unplugged level you should
     # continue on that sequence instead of skipping to next stage)
-    level_to_follow = valid_progression_level? ? next_progression_level : next_level
+    level_to_follow = unplugged? ? next_level : next_progression_level(user)
```

Comment:
Doesn't switching from `valid_progression_level?` to `unplugged?` change the behavior for spelling bee levels? E.g. if you wind up on a spelling bee level as a non-English user, this will return the first level in the next stage, whereas before it would just return the next spelling bee level.

Am I reading this right, and was that an intentional change?